### ISSUE TITLE
Unable to log in, check credentials

### ISSUE THREAD

**Issue by rozhkovets** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#170]
Recently I noticed that the integration stopped working.  The logs contain the error "Unable to log in, check credentials".  But I have not changed my username and password, and I am absolutely sure that the username and password in the integration settings are correct.  Is it possible to somehow find out in more detail what exactly the reason for the failed login is

**Comment by PiotrMachowski** [comment]
Have you tried restarting HA?

**Comment by rozhkovets** [comment]
Sure. I also change password and use new password in integration, also reinstall integration. Login still unsuccessful

**Comment by PiotrMachowski** [comment]
Does [token extractor](https://github.com/PiotrMachowski/Xiaomi-cloud-tokens-extractor) work?

**Comment by rozhkovets** [comment]
Yes, I have now checked and the token received through the extractor matches the one specified in the integration settings

**Comment by PiotrMachowski** [comment]
Have you introduced any changes to your setup/network? (cloudflare dns, adguard, pihole, etc.)

**Comment by rozhkovets** [comment]
I did not make any changes with the network. 

Well, it was a very strange problem.  it was not caused by a misconfiguration of the integration or its operation, but may have been indirectly related to it.

i am using hassio in virtualbox. I installed another HA in parallel.  on this new pure HA, I manually installed the integration and transferred the settings from the problematic HA. and the integration worked without errors.

it looks like it's time for the purge and move to a new ha.
